### ISSUE TITLE
Java.sql.timestamp - BinaryHandlerTest failed

### ISSUE THREAD

**Issue by zdenek-jonas** [microstream-one/microstream#105]
When I save java.sql.Timestamp with actual date and load it back, is the value from this Type lost. 

```
sqlTimestamp ==> expected: <2020-04-07 10:10:14.282> but was: <1970-01-01 01:00:00.282>
Comparison Failure: 
Expected :2020-04-07 10:10:14.282
Actual   :1970-01-01 01:00:00.282
```

**Comment by tm-ms** [comment]
BinaryHandlerSqlTimestamp was created via copy&paste from BinaryHandlerSqlDate, but then not adjusted to the actual type SqlTimestamp.
I will fix this.

**Comment by tm-ms** [comment]
Oh. Funny. SqlTimestamp is a very hot competitor for the title "dumbest class in the JDK".

Besides the long timestamp from the super class, it has an addition int field called "nanos". But it still only accepts a single long in the constructor in MILLISECONDS precision. The constructor splits the passed long value into whole seconds and the remaining fraction of a second. The prior is then passed to the super class constructor (breaking the super class contract, breaking equality comparisons, moreless doing everything wrong that can be done wrong) and the latter is stored in the "nanos" value. Why they did that is beyond me. There is no apparent advantage and the JavaDoc does not provide any rationale, either. It only talks about some of the problems this class has.

Sometimes, the idiocy in the JDK is hardly bearable ...

Bottom line: it still only contains single long value holding milliseconds, but it is internally spread over 12 bytes instead of just 8 while ruining the contracts of the super class's methods in the process.

So while this type has 12 byte state, it must still be enough to just store a 8 byte long value as defined by its API and JavaDoc.

**Comment by tm-ms** [comment]
Fixed, tested and committed directly to the master branch.
Test class is `MainTestStoreSqlTimestamp`.

**Comment by zdenek-jonas** [comment]
Ok, retest ok
